shared/test/resources/tests.yaml

```diff
@@ -1808,3 +1808,10 @@
 "ÞỊ":
   - {in: [[0, 1, 0, 1]], out: [1, 3] }
   - {in: [5], out: [0]}
+
+"øA":
+  - {in: [7], out: "g"}
```

Comment:
Would be nice to also test a number greater than 2^32, i.e., something that doesn't fit in `Int`.